extensions/systemd/index.js

```diff
@@ -22,18 +22,28 @@ class SystemdExtension extends Extension {
                 }
 
                 return false;
+            },
+            onUserSkip: ({instance, ui}) => {
+                ui.log('Systemd setup skipped, reverting to local process manager', 'yellow');
+                instance.config.set('process', 'local').save();
             }
         }];
     }
 
-    _setup({instance}, task) {
+    _setup({instance, ui}, task) {
         const uid = getUid(instance.dir);
 
         // getUid returns either the uid or null
         if (!uid) {
             return task.skip('The "ghost" user has not been created, try running `ghost setup linux-user` first');
         }
 
+        if (instance.config.get('process') !== 'systemd') {
```

Comment:
we might need to change the enable function as well, since it checks if process is systemd

Reply:
What could we do for that? Just `!argv.local`? Or `!argv.local || argv.force` 🤔

Reply:
maybe check if `argv.stages` contains systemd? https://github.com/TryGhost/Ghost-CLI/blob/master/lib/commands/setup.js#L127-L129

Reply:
Can't suggest a patch, but were you thinking this?

```js
enabled: ({instance, argv}) => !argv.local && (instance.config.get('process') === 'systemd' || argv.stages.includes('systemd')),
```

Or completely getting rid of the `instance.config` check?

Reply:
probably
```js
enabled: ({instance, argv}) => !argv.local && (instance.config.get('process') === 'systemd' || (argv.stages && argv.stages.includes('systemd'))),
```

Reply:
sidenote, can't wait until optional chaining is available in node LTS 😂

Reply:
If we switch to typescript we could do it today 😛